02-temperature_converter/temperature.java

```diff
@@ -0,0 +1,21 @@
+import java.util.*;
+
+public class temperature {
+  public static void main(String[] args) {
+    System.out.println("Hello! Input a temperature of Celsius so I can convert it to Farenheit!");
+    celsius();
```

Comment:
now that you know about methods that take in parameters and can return, maybe you can revise this!